Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/db/DatabaseClient.java

```diff
@@ -77,6 +59,18 @@ public ConceptEntry getEntry(String id) {
     }
 
     @Override
+    public ConceptEntry getWrapperEntryByWordnetId(String wordnetId) {
```

Comment:
this and the next method seem to try to do the same thing. As wordnet ids can be multiple and therefore the ```getConceptByWordnetId``` doesn't really work, those two methods should be consolidated.